src/test/endpoints.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Do you mind explaining how this file works? 😅  I've never used postman in this way but looks simple and easy! 

On top of this, could we add tests into the respective Controller test files? You'll likely need to research how to use the testing DB, and how to setup mocks

Basic pattern to the ones [here](https://github.com/ubclaunchpad/PromoPal-backend/blob/master/src/test/controller/UserController.test.ts)

Reply:
Could you check the "How" section of my PR description? Please let me know if anything is unclear.
I don't think writing tests are the priority now? I may add automated tests now, but for now I think manual testing may suffice.